iModelCore/iModelPlatform/DgnCore/DgnUnits.cpp

```diff
@@ -5,6 +5,26 @@
 #include <DgnPlatformInternal.h>
 #include <DgnPlatform/DgnGeoCoord.h>
 
+/*---------------------------------------------------------------------------------**//**
+* @bsimethod
++---------------+---------------+---------------+---------------+---------------+------*/
+StatusInt EcefLocation::ComputeTransform(TransformR ecefTrans)

```

Comment:
```suggestion
Transform EcefLocation::ComputeTransform()
```

You only ever return `SUCCESS` anyway. Just return a transform directly.